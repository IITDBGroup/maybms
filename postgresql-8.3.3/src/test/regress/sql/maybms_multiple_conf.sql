--test for multiple conf in a query

create table r (type varchar, p int);
insert into r values('a', 1), ('a', 2), ('b', 1), ('b', 2);

create table C as repair key type in r weight by p; 

select type, conf() from C group by type;

select type, conf(), conf() from C group by type;

select type, conf(), aconf(.5,.5) from C group by type;

select conf(), conf() from C;

drop table r;
drop table C;
