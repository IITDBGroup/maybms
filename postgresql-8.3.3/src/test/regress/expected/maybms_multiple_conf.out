--test for multiple conf in a query
create table r (type varchar, p int);
insert into r values('a', 1), ('a', 2), ('b', 1), ('b', 2);
create table C as repair key type in r weight by p; 
select type, conf() from C group by type;
 type | conf 
------+------
 a    |    1
 b    |    1
(2 rows)

select type, conf(), conf() from C group by type;
ERROR:  Query not supported: multiple approximate aggregation nodes not supported.
select type, conf(), aconf(.5,.5) from C group by type;
ERROR:  Query not supported: multiple approximate aggregation nodes not supported.
select conf(), conf() from C;
ERROR:  Query not supported: multiple approximate aggregation nodes not supported.
drop table r;
drop table C;
